tempesta_fw/sock_srv.c

```diff
@@ -143,6 +150,7 @@ tfw_sock_srv_connect_try(TfwSrvConn *srv_conn)
 	                   &sk);
 	if (r) {
 		TFW_ERR("Unable to create server socket\n");
+		tfw_srv_conn_stop(srv_conn);
```

Comment:
If we exit here on error from `ss_sock_create()`, there will be no more calls to `tfw_sock_srv_connect_try()` since the timer is expired (and, consequently, deactivated) and not rearmed (as e.g. below - after `ss_connect()` error). So, in fact - the connection will be stopped here and we need to set appropriate bit and put the server. This behavior provides determination of all states in which connection is allowed to be, and support consistency with the waiting cycle in `tfw_sock_srv_disconnect()` (as described in [comment](https://github.com/tempesta-tech/tempesta/blob/0fc370f5d3cc47df513dab7dbe214fd46eb12c8c/tempesta_fw/sock_srv.c#L451)).

Reply:
Seems like the error may happen at Nth reconnect attempt, and connection will never be connected again.

Reply:
Added reconnecting on OOM.